scripts/src/docker/ethereum.rs

```diff
@@ -121,16 +121,16 @@ impl Account {
     }
 }
 
-async fn fund_new_account(endpoint: ParityHttpEndpoint) -> anyhow::Result<Account> {
-    let (_event_loop_handle, transport) = Http::new(&endpoint.to_string())
+async fn fund_new_account(endpoint: GethHttpEndpoint) -> anyhow::Result<Account> {
+    let transport = Http::new(&endpoint.to_string())
```

Comment:
OMG did they finally fix this? I can't believe it!

Reply:
:)